crates/m3/src/builder/constraint_system.rs

```diff
@@ -182,7 +182,7 @@ impl<F: TowerField> ConstraintSystem<F> {
 			if count == 0 {
 				continue;
 			}
-			if table.power_of_two_sized {
+			if table.is_power_of_two_sized() {
```

Comment:
Curious: why won't we assert that if the table is of fixed log size, then `count` should actually equal to that size?